### DISCUSSION TITLE
BlazorWebAssemblySupabaseTemplate

### DISCUSSION THREAD

**Discussion by BasBBakker** [supabase-community/supabase-csharp#70] [General]
Can you update the BlazorWebAssemblySupabaseTemplate example to work with the new version of GoTrue?

I cannot get the LoadSession to retrieve a stored session from localStorage.

Thanks so much in advance!

Bas

**Comment by acupofjose** [comment]
@BasBBakker sorry for the delay - I've update the project to support up to v0.11.0! The file in question is [here](https://github.com/supabase-community/supabase-csharp/blob/master/Examples/BlazorWebAssemblySupabaseTemplate/Providers/CustomAuthStateProvider.cs).

**Comment by BasBBakker** [comment]
You are awfully fast in all your replies! Thank you very much!

**Comment by BasBBakker** [comment]
I add  the following function:

 private async void LoadSession()
    {

        client.Auth.LoadSession();
        await client.Auth.RetrieveSessionAsync();

    }

the line client.Auth.LoadSession() gives an error:


<img width="797" alt="image" src="https://github.com/supabase-community/supabase-csharp/assets/25266030/1194ed40-ddeb-436d-8898-d59149fb7263">

Would you know how to avoid this error? The example does not seem to use LoadSession().

Thanks for your help!

Bas
